### PR TITLE
Added Xcode 8 (Beta 6) support

### PR DESCRIPTION
**Pull request by mohpor**
Hi,

I have added Xcode 8 (Beta 6) support to the library.

Sincerely,
M.

### CODE REVIEW

**Comment by m-revetria** [comment]
@mohpor Thanks for your contribution, I'll check it out

File: Example/CustomActionControllers/Periscope.swift [hunk 1]
```diff
@@ -78,28 +78,31 @@ public class PeriscopeHeader: UICollectionReusableView {
...
-        fatalError("init(coder:) has not been implemented")
+        super.init(coder: aDecoder)
     }
 }
 
 public class PeriscopeActionController: ActionController<PeriscopeCell, String, PeriscopeHeader, String, UICollectionReusableView, Void> {
-    
-    public override init(nibName nibNameOrNil: String? = nil, bundle nibBundleOrNil: NSBundle? = nil) {
+  
+  required public init?(coder aDecoder: NSCoder) {
```
Comment (m-revetria) [inline Example/CustomActionControllers/Periscope.swift]: please indent using 4 spaces

File: Example/CustomActionControllers/Periscope.swift [hunk 2]
```diff
@@ -78,28 +78,31 @@ public class PeriscopeHeader: UICollectionReusableView {
...
+    super.init(coder: aDecoder)
+  }
```
Comment (m-revetria) [inline Example/CustomActionControllers/Periscope.swift]: add a blank separator line here

File: Example/CustomActionControllers/Periscope.swift [hunk 3]
```diff
@@ -108,19 +111,21 @@ public class PeriscopeActionController: ActionController<PeriscopeCell, String,
         
         onConfigureHeader = { [weak self] header, headerData in
             guard let me = self else { return }
-            header.label.frame = CGRectMake(0, 0, me.view.frame.size.width - 40, CGFloat.max)
+          header.label.frame = CGRect(x: 0,y: 0, width: me.view.frame.size.width - 40, height: CGFloat.greatestFiniteMagnitude)
```
Comment (m-revetria) [inline Example/CustomActionControllers/Periscope.swift]: fix indentation here

File: Example/CustomActionControllers/Skype.swift [hunk 4]
```diff
@@ -174,9 +179,9 @@ public class SkypeActionController: ActionController<SkypeCell, String, UICollec
                 me.contextView.diff = -pixels
                 me.contextView.setNeedsDisplay()
                 
-                if self?.collectionView.frame.origin.y > self?.view.bounds.size.height {
+                if self!.collectionView.frame.origin.y > self!.view.bounds.size.height {
```
Comment (m-revetria) [inline Example/CustomActionControllers/Skype.swift]: please don't use forced unwrapping. Here you can use (we should have used) the local variable `me`

File: Example/CustomActionControllers/Spotify.swift [hunk 5]
```diff
@@ -128,53 +128,56 @@ public class SpotifyHeaderView: UICollectionReusableView {
...
-        blurView.autoresizingMask = UIViewAutoresizing.FlexibleHeight.union(.FlexibleWidth)
+        let blurView = UIVisualEffectView(effect: UIBlurEffect(style: .dark))
+        blurView.autoresizingMask = [.flexibleHeight, .flexibleWidth]
         return blurView
     }()
-    
+  
+    required public init?(coder aDecoder: NSCoder) {
+      super.init(coder: aDecoder)
+    }
```
Comment (m-revetria) [inline Example/CustomActionControllers/Spotify.swift]: please add a blank line

File: Source/ActionController.swift [hunk 6]
```diff
@@ -663,46 +684,46 @@ public class DynamicsActionController<ActionViewType: UICollectionViewCell, Acti
     
     // MARK: - UICollectionViewDelegate & UICollectionViewDelegateFlowLayout
     
-    public override func collectionView(collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAtIndexPath indexPath: NSIndexPath) -> CGSize {
-        guard let alignment = (collectionViewLayout as? DynamicCollectionViewFlowLayout)?.itemsAligment where alignment != .Fill else {
-            return super.collectionView(collectionView, layout: collectionViewLayout, sizeForItemAtIndexPath: indexPath)
+    open override func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
+        guard let alignment = (collectionViewLayout as? DynamicCollectionViewFlowLayout)?.itemsAligment , alignment != .fill else {
```
Comment (m-revetria) [inline Source/ActionController.swift]: please remove white space before comma in `itemsAligment , aligment`

File: Source/ActionController.swift [hunk 7]
```diff
@@ -24,17 +24,37 @@
 
 import Foundation
 import UIKit
+fileprivate func < <T : Comparable>(lhs: T?, rhs: T?) -> Bool {
```
Comment (m-revetria) [inline Source/ActionController.swift]: where are these functions used?

**Comment by m-revetria** [comment]
@mohpor, I just finished reviewing your PR and I left some simple comments.

Please rebase latest changes and fix the comments and we will be very glad to merge this.

**Comment by mohpor** [comment]
@m-revetria Thanks for the review.
I guess in a few hours the Xcode 8.0 Final will be out there. I'm going to rebase and apply your comments once it is released.

**Comment by m-revetria** [comment]
@mohpor yeah, it makes absolutely sense.

Thank you!

**Comment by mohpor** [comment]
I'm going to close this PR in favor of #24

**Pull request closed without merging**